packages/nextjs/app/builders/0x8Bb563DC969cC3b3e7275474c3847b44dBCB9441/page.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import Image from "next/image";
+import { NextPage } from "next";
+
+const MichaelEsenwa: NextPage = () => {
+  const contactLink = [
+    {
+      link: "https://wa.me/2347081293274?text=Hello Michael",
+      image: "https://img.shields.io/badge/WHATSAPP-%2325D366.svg?&style=for-the-badge&logo=whatsapp&logoColor=white",
```

Comment:
Instead of using external links to images you could use a library like `react-icon` which is already installed